aemedge/templates/article/article.js

```diff
@@ -12,6 +12,29 @@ function decorate(doc) {
     main.insertBefore(tocSection, doc.querySelector('main > :nth-child(2)'));
   }
 
+  // Wrap images and captions in common container for sizing
+  const pictures = doc.querySelectorAll('main > div > p > picture');
+  pictures.forEach((pictureEl) => {
+    const parent = pictureEl.parentElement;
+    const { nextElementSibling } = parent;
+    const isCaption = nextElementSibling?.tagName.toLowerCase() === 'p'
```

Comment:
Quick comment: we can remove `.toLowerCase()` and keep `tagName === 'P'` as it always  returns a fixed value